wp-content/themes/pub/wporg-learn-2020/template-parts/component-post-card.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * The template part for display a post card
+ *
+ * @package WordPressdotorg\Theme;
+ */
+
+namespace WordPressdotorg\Theme;
+
+$args = wp_parse_args( $args );
+$post_type_name = get_post_type();
+$post_type_info = get_post_type_object( $post_type_name );
+
+?>
+<article class="post-card">
+	<?php if ( ! in_array( $post_type_name, array( 'page', 'attachment', 'post' ) ) ) : ?>
+	<div class="post-card_tag post-card_tag--<?php echo esc_attr( $post_type_name ); ?>">
+		<?php echo esc_attr( $post_type_info->labels->singular_name ); ?>
+	</div>
+	<?php endif; ?>
+	<div class="entry">
+		<header class="entry-header">
+			<h3 class="entry-title">
+				<a href="<?php echo esc_url( get_permalink() ); ?>" rel="bookmark"><?php the_title(); ?></a>
+			</h3>		
+		</header><!-- .entry-header -->
+
+		<div class="entry-excerpt">
+			<p><?php the_excerpt(); ?></p>
+		</div><!-- .entry-excerpt -->
+	</div>		
+	<?php
+	// If we don't have a template, don't show the footer
+	get_template_part( 'template-parts/component', 'post-card-footer-' . $post_type_name );
```

Comment:
Will there be different footers for other post types? why not put this behind another `if ( workshop === $post_type_name )` check? Right now, if someone creates `component.php`, that'll show in the footer as a fallback due to how the template loader works.

An alternative could be this, and create `template-parts/component-post-card-footer.php` as an empty file.
```suggestion
	get_template_part( 'template-parts/component-post-card-footer', $post_type_name );
```

Reply:
Good call. There should be a footer for lesson plans, this PR doesn't implement it because the metadata is not added in production but I see a future where we fix that. We are also missing "workshop ideas" which isn't implemented yet ( and may never be since we have workshops now). But we could potentially end up with 3 different footers.